sds_data_manager/utils/stackbuilder.py

```diff
@@ -194,23 +195,21 @@ def build_sds(
     )
 
     # This valid instrument list is from imap-data-access package
-    for instrument in imap_data_access.VALID_INSTRUMENTS:
-        ecr = ecr_construct.EcrConstruct(
-            scope=sdc_stack,
-            construct_id=f"{instrument}Ecr",
-            instrument_name=f"{instrument}",
-        )
-
-        batch_compute_resources.FargateBatchResources(
-            scope=sdc_stack,
-            construct_id=f"{instrument}BatchJob",
-            vpc=networking.vpc,
-            processing_step_name=instrument,
-            data_bucket=data_bucket.data_bucket,
-            repo=ecr.container_repo,
-            db_secret_name=db_secret_name,
-            efs_instance=efs_instance,
+    processing_volumes = [
+        batch.EfsVolume(
+            name=f"{efs_instance.volume_name}-ECS-mount",
+            access_point_id=efs_instance.spice_access_point.access_point_id,
+            file_system=efs_instance.efs,
+            container_path="/mnt/spice",
```

Comment:
can you also add mountPoints? I believe that mounts efs onto batch volume.

Reply:
This is the mount point. Inside the container this EFS drive is mounted at `/mnt/spice`